src/test/java/basics/Exercise1Test.java

```diff
@@ -0,0 +1,16 @@
+package basics;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class Exercise1Test {
+    @Test
+    public void test_gcd(){
+        assertEquals(new Exercise1().gcd(3, 2), 1);
+    }
+    @Test
+    public void test_lcm(){
+        assertEquals(new Exercise1().lcm(3, 2), 6);
```

Comment:
Add more cases because `3, 2, 6` is a very special case `2*3=6`

Reply:
Done